### PR TITLE
feat: add pools component

### PR DESCRIPTION
**Pull request by MishaZhem**
Closes #

### CODE REVIEW

File: apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                    class="viewport tui-zero-scrollbar"
+                >
+                    <div *cdkVirtualFor="let i of lengthPools(data.length)">
+                        <div
+                            *ngIf="
+                                (data[i].symbol.toUpperCase() + '/' + data[i + 1].symbol.toUpperCase()).includes(
+                                    search.toUpperCase()
+                                )
+                            "
+                            tuiCell="m"
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.html:48]: let's use table

File: apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                        [columns]="columns"
+                    >
+                        <thead>
+                            <tr tuiThGroup>
+                                @for (column of columns; track $index) {
+                                    <th
+                                        *tuiHead="column"
+                                        tuiTh
+                                        [sticky]="true"
+                                        [style.top.px]="-viewport['_renderedContentOffset']"
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.html:41]: We shouldn't use private properties. Use public method `-(viewport.getOffsetToRenderedContentStart()  || 0)`

File: apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                                *cdkVirtualFor="let item of tableData() | tuiTableSort"
+                                tuiTr
+                            >
+                                @for (column of columns; track $index) {
+                                    <td
+                                        *tuiCell="column"
+                                        tuiTd
+                                        [style.padding-left]="'0.5rem'"
+                                    >
+                                        @if (column === 'Pair') {
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.html:59]: Unnecessary cycle with conditions. Each row contains three cells, so we can just make template without conditions

File: apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+                                                        [src]="
+                                                            'https://assets.coincap.io/assets/icons/' +
+                                                            item.symbolFirst +
+                                                            '@2x.png'
+                                                        "
+                                                    />
+                                                    <tui-avatar
+                                                        size="m"
+                                                        [src]="
+                                                            'https://assets.coincap.io/assets/icons/' +
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.html:76]: we use this in a lot of crypto components
let's make pipe `coinIcon` and will use it like `[src]="item.symbolSecond | coinIcon"`

File: apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.ts [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    ],
+    templateUrl: './pools.component.html',
+    styleUrl: './pools.component.less',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PoolsComponent {
+    protected cryptoService = inject(CryptoService);
+    protected tokens = toSignal(this.cryptoService.getTokens());
+    protected tableData: Signal<TableData[]> = computed(() =>
+        (this.tokens() ?? [])
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.ts:62]:
```
      const tokens = this.tokens() || [];

        return tokens
            .map((_, index) => ({
                Pair: `${tokens?.[index]?.symbol.toUpperCase()}/${tokens?.[index + 1]?.symbol.toUpperCase()}`,
 ...

**Review by vladimirpotekhin — DISMISSED** [review]

File: apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.html [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+                Pools
+            </h2>
+        </header>
+        <tui-input
+            tuiTextfieldIconLeft="@tui.search"
+            [(ngModel)]="search"
+        >
+            Search
+        </tui-input>
+        <div class="list-pools">
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.html:22]: lets make a little bigger

File: apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.ts (hunk 5 above)
Comment (MishaZhem) [reply apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.ts:62]: constant will be equal to null then

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**